### PR TITLE
Profile css

### PR DESCRIPTION
**Pull request by ChristianPrevitali**
profile css and history overlap bug fix

### CODE REVIEW

File: src/components/Accounts.js [hunk 1]
```diff
@@ -60,27 +53,29 @@ const Account = ({ user }) => {
...
-                  </button>
-                  <br></br>
-                  <button onClick={sOut}>Sign Out</button>
-                  <br></br>
-                  <button onClick={() => { setShow(true) }}>Delete Account</button>
-                  <Form onSubmit={handleSubmit}>
-                    <Button className="w-40 mt-10" type="submit">
-                      Home
+
+                  <div id = 'btn'>
```
Comment (BaconB0T) [inline src/components/Accounts.js:59]: ids should be unique. Refactor into class

Comment (BaconB0T) [reply src/components/Accounts.js:59]: with a different name, bootstrap has classes with these names that we use in various places around the app so name them something else

File: src/styles/Accounts.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#btn{
```
Comment (BaconB0T) [inline src/styles/Accounts.css:1]: See prev. comment about IDs and classes

**Review by BaconB0T — CHANGES_REQUESTED** [review]
See comments. IDs on HTML elements should be unique, so refactor shared IDs into classes, and make new names for them to avoid overwriting the bootstrap classes

File: src/components/Accounts.js (hunk 1 above)
Comment (ChristianPrevitali) [reply src/components/Accounts.js:59]: how about you make me

**Review by BaconB0T — APPROVED** [review]
👍

**Pull request merged**